### DISCUSSION TITLE
Can we "host" the discussions forum usder our own custom domain?

### DISCUSSION THREAD

**Discussion by Shankar1598** [community/community#54317] [Discussions] [answered]
### Select Topic Area

Product Feedback

### Body

Let's say I have the domain example.com, It will be nice to have one of these features:

1. Point discuss.example.com domain to "github.com/orgs/example_org/discussions"
2. Point example.github.com domain to "github.com/orgs/example_org/"

**Comment by ichenglin** [comment, marked as answer]
You can't directly serve GitHub discussion under your domain,
however it is possible for you to redirect users instead.

To redirect users, you could either set a redirect rule from your domain provider,
or have your web server return a redirect request.

**Reply by Docdewy** [reply]
Here is proof of me being overrides.

**Reply by Docdewy** [reply]
I am needing a complete account and computer overview. Would like a phone number to talk to someone. Also, I'm under 24/7 stalking, MeControl and he is running a copied  Parent Computer out of Pierce County Library System. Is running a dual line and vendor through friends phones. Collecting my dividends in Library and running a scam, that 37countries are after. I'll make a reward offer for the group or business, that can track in my account, he is using my Shareholders rights and finances. Microsoft
Is also not seeing this. I authorize you look for

**Reply by Docdewy** [reply]
How he is doing it and tell, Satya Nadella. Duane Parsons is allowing him the same rights, to see illegal Internal Control, redirection of money, mail and the 29 years of being overrided and all my investments redirected to a fraud account and company, Enterprise...

**Reply by Docdewy** [reply]
Tell Satya, " RINGS A BELL!" FROM ME...

**Reply by Docdewy** [reply]
> You can't directly serve GitHub discussion under your domain, however it is possible for you to redirect users instead.
> 
> To redirect users, you could either set a redirect rule from your domain provider, or have your web server return a redirect request.

No, you can't, There is a reason why! Besides you authorizing it, what are you authorizing?

**Comment by laymonage** [comment]
Not sure if it's what you're looking for, but you could try https://giscus.app which lets you embed a GitHub Discussion on any website :)
